mqemitter-mongodb.js

```diff
@@ -30,7 +31,7 @@ function MQEmitterMongoDB (opts) {
 
   if (opts.db) {
     that._db = opts.db
-    waitStartup()
+    setImmediate(waitStartup)
```

Comment:
This is needed because without this a user could not subscribe to `error` event that could be thrown before the event listener is setup